src/pybind/mgr/dashboard/controllers/iscsi.py

```diff
@@ -26,12 +26,18 @@ class IscsiUi(BaseController):
     @Endpoint()
     @ReadPermission
     def status(self):
+        required_ceph_iscsi_config_version = 6
```

Comment:
Can we make this a static attribute of the iSCSI controller class with all caps? to make it more visible...

Reply:
@rjfd Done